Strategies/IITB_KPIs.md

```diff
@@ -0,0 +1,74 @@
+
+- Focus on the Core Business (of ESDC)
+- Modernize IM/IT Service Delivery
+- Be efficient in managing its operations
+- Be connected with IM/IT partners and suppliers
+
+## 1. **Percentage of ESDC Services consumable by APIs**
+
+ESDC services provided to the public or to other departments will need to be delivered digitally. This means via APIs. This transformation to digital will be measured by the rate at which analogue services are changed to digital ones.
+
+The indicator will be measured by using [ESDC's Service Catalogue](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) as reference point and monitor the rate at which they are transformed to digital
+
+`percentage = digital services / total services`
+
+## 2. **Rate of decrease in processing time of ESDC Services**
+
+The processing time of ESDC services that are provided to the public or to other departments will be an indication of improvement. The measurement will be the reduction in time, not the time itself, focusing on continuous improvement rather than a hard target.
+
+The indicator will be measured by using [ESDC's Service Catalogue](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) and monitor their total processing time from beginning to end. This amount, over time, will be monitored to indicate reduction in processing time.
+
+`rate of service = (New Processing Time - Old Processing Time) / Old Processing Time`
+
+## 3. **Reduction in toil of IITB and ESDC Services**
+
+Toil is the kind of work tied to running a production service that tends to be manual, repetitive, automatable, tactical devoid of enduring value, and that scales linearly as a service grows. Processing  time of ESDC services may decrease by increasing manual labour but that would defeat the purpose of digital.
+
+The indicator will be measured two ways:
+
+1. For toil in IITB Services, using ??? and measure the ratio of X over Y. 
+
+`ratio = X / Y`
+
+2. For toil in ESDC Services, using the [ESDC's Service Catalogue](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) and measure the ratio of service requests to amount of personnel. 
+
+`ratio = service requests / service personnel`
+
+## 4. **Percentage of IITB services made available as SaaS**
+
+IITB services currently offered, as catalogued in the ITSM Repository, will be offered as SaaS for IITB consumers. This transformation to SaaS provisioning and maintenance will be measured by the rate at which those services are changed to self-served ones. The measurement will look at [IITB's Service Catalogue](http://iservice.prv/eng/service_catalogues/imit_services.shtml), which is being modernized for 2020, as reference point and monitor their transformation to SaaS.
+
+`percentage = SaaS / total ITSM services`
+
+## 5. **Availability uptime of ESDC core systems**
```

Comment:
That's a good KPI but I'm wondering if we can use it since we're not really in control of anything? For example, we could measure this current KPI and measure them IF our strategies are put in place.

Reply:
All the indicators here are meant to be used IF our strategies are put in place.

This one is to push towards a service approach in that the complexity and growth of a service is of no concern to its consumer. Currently, our "service" is through BRDs and personnel planning. The complexity of our "service" is a concern to our consumer because they have to plan according to our demand. 

This is the fear that some have. E.g.putting an API on the API store "How will we manage growth? What if we open the flood gate and we can't keep up? What if what if what if ??" Service uptime is of the responsibility of the service owner and they'll need to handle that complexity. Cloud is great for its rapid elasticity, DevOps is great for rapid response to change, etc.

Reply:
Sounds good!